site/en/docs/privacy-sandbox/third-party-cookie-phase-out/index.md

```diff
@@ -83,12 +84,18 @@ Set-Cookie: cookie-name=value; SameSite=None; Secure
 Make sure to review your cookies and have a list of those set with the `SameSite=None`. These are the cookies for which you will need to take action to ensure they keep functioning properly.
 {% endAside %}
 
-One way to identify them is to examine your code base and search for cookies containing `SameSite=None`.
+## Debugging cookies
+
+One way to identify third-party cookies in your code base is to search for cookies containing `SameSite=None`.
 
 Another option is to browse through your site with third-party cookies blocked on your machine and use DevTools to investigate any potential breakage.
 
+You can block third-party cookies from Chrome settings or by [running Chrome from the command line](https://www.chromium.org/developers/how-tos/run-chromium-with-flags/), version 115 or higher, with the flag `--test-third-party-cookies-phaseout` or `–-enabled-features=ForceThirdPartyCookieBlocking`.
```

Comment:
```suggestion
In Chrome version 115 or higher, you can test the browser behavior after third-party cookie phase out by [running Chrome from the command line](https://www.chromium.org/developers/how-tos/run-chromium-with-flags/) with the flag `--test-third-party-cookies-phaseout` or `–-enabled-features=ForceThirdPartyCookieBlocking`. This will block third-party cookies, and also enable third-party storage partitioning and [First-Party Sets](/docs/privacy-sandbox/first-party-sets).
```